serving/src/main/java/ai/djl/serving/util/ConfigManager.java

```diff
@@ -238,8 +239,10 @@ public Connector getConnector(Connector.ConnectorType type) {
         String binding;
         if (type == Connector.ConnectorType.MANAGEMENT) {
             binding = prop.getProperty(MANAGEMENT_ADDRESS, "http://127.0.0.1:8080");
-        } else {
+        } else if (type == Connector.ConnectorType.INFERENCE) {
             binding = prop.getProperty(INFERENCE_ADDRESS, "http://127.0.0.1:8080");
+        } else {
+            binding = prop.getProperty(CLUSTER_ADDRESS, "http://127.0.0.1:8888");
```

Comment:
This isn't correct for `ConnectorType.BOTH`

Reply:
I will change to switch case to make it more readable